.github/workflows/deploy.yml

```diff
@@ -1,7 +1,11 @@
 name: Deploy
 
 # Run on every commit
-on: push
```

Comment:
starting from the latest develop branch would prevent such issues

Reply:
I'll paste what I wrote in the other PR:

@maria-vslvn when starting a new ticket in a new branch I'd suggest to do the following so you don't pick the wrong commits as with the `renaming deployments` which prevents Github Actions to run the deployment script:

```bash
git fetch
git checkout -b feature/#479-mob-menu-issue origin/develop
```

This will ensure to always fetch the latest changes from origin and then create the branch from the latest commit.